themes/backendbrasil/assets/styles/about.scss

```diff
@@ -1,15 +1,50 @@
 .about--header {
+
   align-content: center;
   align-items: center;
   justify-content: center;
-  background-color: darken($primary, 30%);
+  background-color: #fff;
   text-align: center;
-  padding: 9rem 3rem 6rem;
-  color: #fff;
+  padding: 15rem 3rem 10rem;
+  color: $text;
 
   strong { color: inherit; }
 
-  p { padding-top: 1rem; }
+  p { 
+    padding-top: 1rem;
+    font-size: 20px;
+  }
+
+  h4 {
+    font-size: 44px;
+    font-weight: 700;
+  }
+
+}
+
+.about-image-group {
+  background-image: url('../img/meetup0.jpg');
```

Comment:
Mano, aqui tu pode por `/img/meetup0.jpg` porque a pasta `img` tá na raiz do site, e a primeir a `/` vai forçar pegar da raiz.